### DISCUSSION TITLE
HELP PLEASE! Webui.sh stops with error msg ERROR: Cannot activate python venv, aborting...

### DISCUSSION THREAD

**Discussion by keith-maltman** [OpenTalker/SadTalker#847] [Q&A]
After a lifetime of using computers I built my first ever Linux (Ubuntu) PC a couple of weeks ago, to write a Python AI. So far so good, but looking ahead trying to find a find potential end for the AI. SadTalker looks very interesting but wont install using your webui.sh script.   Prob something silly but I have now spent the entire day trying to gets this to work (having only a couple of weeks Linux experience  does not help).

The current error is
 
ERROR: Cannot activate python venv, aborting...

Any logical approach or tips or suggestions would be very helpful.

Keith

**Comment by danka12324** [comment]
I'm sorry, but I can't provide you with detailed information on how to solve this problem because I don't have access to your computer and can't see all the necessary details. However, I can offer some general tips that can help you figure out the problem.:

Make sure that you have installed all the necessary dependencies for the script to work. Check the list of dependencies and install them if they are missing.
Try running the script as an administrator. This may be required to perform some operations, such as creating a virtual environment.
Check if you have the rights to execute the script. You may need to change the rights to the script file.
If nothing helps, try to find alternative ways to install SadTalker, for example, through the pip package manager or through the Ubuntu GUI.
If these general tips do not help, it may be worth seeking help from the Ubuntu developer community or the authors of a particular project
